### PR TITLE
Compose rework

### PR DESCRIPTION
**Pull request by Ducktatorrr**
- Removed equal signs from enviornment variables. This allows Docker Compose to pickup the values from the .env

- Added DEVNET_API_KEY to .env.example and added some comments

### CODE REVIEW

**Comment by shawaj** [comment]
@Ducktatorrr the =none removal will break things unless you update the python script too

**Comment by shawaj** [comment]
Actually - maybe not as I already added a default here https://github.com/Erwin-Schrodinger-Token/ewn-box-opener/blob/main/run_box_opener.py

But maybe need to test

**Comment by Ducktatorrr** [comment]
yeah, it's being set to "none" if not provided: https://github.com/Erwin-Schrodinger-Token/ewn-box-opener/blob/main/run_box_opener.py#L16

Awaiting your confirmation

**Comment by Ducktatorrr** [comment]
Screenshot 2024-10-23 at 11 38 42

Looks ok, right?

**Comment by Ducktatorrr** [comment]
Screenshot 2024-10-23 at 12 03 55

Looks ok, right? It does keep looking for it but that also happened before.

**Comment by shawaj** [comment]
> Screenshot 2024-10-23 at 12 03 55
> Looks ok, right? It does keep looking for it but that also happened before.

Yeah, it looks fine to me. The none is only used to determine whether it is set at all or not

**Review by BFGNeil — COMMENTED** [review]
LGTM

**Comment by shawaj** [comment]
Hurry up @robputt 🤪

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,12 @@
-version: '2.4'
```
Comment (shawaj) [inline docker-compose.yml:1]: Will need to keep this in for balena

**Comment by Ducktatorrr** [comment]
Ammended @shawaj

**Review by robputt — APPROVED** [review]

**Pull request merged**